ClientServer.go

```diff
@@ -47,7 +46,10 @@ func Server() {
 		// send new string back to client
 		_, sendingerror := conn.Write([]byte(newmessage + "\n"))
```

Comment:
as I have already used variable 'err' above in this loop where I have used 'sendingerror' and if I use 'err' for this line again then compiler could not differentiate these two variables having same name.